### PR TITLE
CONSOLE-2361: Remove orphaned Bootstrap and PatternFly styles

### PR DESCRIPTION
**Pull request by rhamilto**

### CODE REVIEW

**Comment by rhamilto** [comment]
/hold for approval

/assign @yapei for QE approval

This PR has no user visible changes, and we have general docs and px approval for this epic. @ahardin-rh @sferich888 FYI

/label docs-approved
/label px-approved

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**
